src/modelgauge/private_ensemble_annotator_set.py

```diff
@@ -52,20 +53,23 @@ def __init__(self, secrets):
         self.__configure_vllm_annotators()
         self.__configure_huggingface_annotators()
         self.__configure_together_annotators()
-        self._load_annotators()
+        self._register_annotators()
 
-    def _load_annotators(self):
-        self.annotators = {
-            MISTRAL_8x22B_CONFIG.name: PromptEngineeredAnnotator(
-                uid=MISTRAL_8x22B_CONFIG.name, config=MISTRAL_8x22B_CONFIG
-            ),
-            LLAMA_3_70B_CONFIG.name: PromptEngineeredAnnotator(uid=LLAMA_3_70B_CONFIG.name, config=LLAMA_3_70B_CONFIG),
-            LG2_LORA_CONFIG.name: LlamaGuard2LoRAAnnotator(uid=LG2_LORA_CONFIG.name, config=LG2_LORA_CONFIG),
-            WILDGUARD_ANNOTATOR_CONFIG.name: WildguardAnnotator(
-                uid=WILDGUARD_ANNOTATOR_CONFIG.name,
-                config=WILDGUARD_ANNOTATOR_CONFIG,
-            ),
-        }
+    def _register_annotators(self):
+        # TODO: Register annotators in secret repo.
```

Comment:
Do you mean the register and configure bits will be handled outside of this class?

Reply:
It's not necessary but I was thinking that at some point we might want WS3 to fully manage the configuration and UIDs of annotator